123_state_tasks.R

```diff
@@ -53,13 +54,20 @@ do_state_tasks <- function(oldest_active_sites, ...) {
     include = 'remake.yml',
     packages = c('tidyverse', 'dataRetrieval' , 'lubridate'),
     sources = c(...),
-    tickquote_combinee_objects = FALSE,
-    finalize_funs = c())
+    final_targets = c('obs_tallies', '3_visualize/out/timeseries_plots.yml'),
+    finalize_funs = c('combine_obs_tallies', 'summarize_timeseries_plots'),
+    as_promises = TRUE,
+    tickquote_combinee_objects = TRUE)
 
   # Build the tasks
-  scmake('123_state_tasks', remake_file='123_state_tasks.yml')
+  obs_tallies <- scmake('obs_tallies_promise', remake_file='123_state_tasks.yml')
+  scmake('timeseries_plots.yml_promise', remake_file='123_state_tasks.yml')
+  #to read timeseriser plots
+  timeseries_plots_info <- yaml::yaml.load_file('3_visualize/out/timeseries_plots.yml') %>%
+    tibble::enframe(name = 'filename', value = 'hash') %>%
+    mutate(hash = purrr::map_chr(hash, `[[`, 1))
   # Return nothing to the parent remake file
-  return()
+  return(list(obs_tallies = obs_tallies, timeseries_plots_info = timeseries_plots_info))
```

Comment:
remember to edit the comment to reflect that now we're returning ___, not nothing, to the parent remake file